tests/dash/app_virtualization.py

```diff
@@ -0,0 +1,27 @@
+import dash_table
+import pandas as pd
+
+# Subset of https://www.irs.gov/pub/irs-soi/16zpallagi.csv
+df = pd.read_csv("./datasets/16zpallagi-25cols-100klines.csv")
```

Comment:
100k lines, 25 cols (+1 row id column) subset of some large IRS file, limited it to 100k lines for demo purposes, original file is 170k and works just fine.. it's just longer to load b/c of the initial FE/BE data transfert.